srcpkgs/hugin/patches/find-openexr-2_5.patch

```diff
@@ -23,7 +23,7 @@
    find_library_with_debug(OPENEXR_ILMTHREAD_LIBRARY 
      WIN32_DEBUG_POSTFIX d
 -    NAMES IlmThread-2_3 IlmThread-2_2 IlmThread
-+    NAMES IlmThread-2_4 IlmThread-2_3 IlmThread-2_2 IlmThread
++    NAMES ilmthread-2_5 ilmthread-2_4 ilmthread-2_3 IlmThread-2_2 IlmThread
```

Comment:
Not sure if this is case sensitive, but match the `IlmThread` capitalization for consistency unless the all-lowercase form is needed to work.